### ISSUE TITLE
AttributeError: 'Evaluator' object has no attribute 'modules'

### ISSUE THREAD

**Issue by 10gic** [statmobile/pydoc#19]
When I run pydoc-at-point, in buffer `*pydoc*` it reports:
Traceback (most recent call last):
  File "/var/folders/3h/s9r7m0xn1s5gp0s8_jrjnv6h0000gn/T/py-504JjY", line 17, in <module>
    script_modules = gd[0]._evaluator.modules
AttributeError: 'Evaluator' object has no attribute 'modules'

In modeline, it show error message:
Invalid regexp: "Unmatched [ or [^"

My python script is:
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import threading

def fun1():
    pass

if __name__ == '__main__':
    t1 = threading.Thread(target=fun1)
    t1.start()
```
My cursor is under `threading` keyword in line 10.

My system env:
```
$ python --version
Python 3.6.5
$ uname
Darwin
```

**Comment by jkitchin** [comment]
I can't reproduce this with Python 3.6.0 also on Darwin.

**Comment by 10gic** [comment]
@jkitchin The internal var python-script in function pydoc-at-point is:
```
import jedi
s = jedi.Script("""#!/usr/bin/env python
# -*- coding: utf-8 -*-

import threading

def fun1():
    pass

if __name__ == '__main__':
    t1 = threading.Thread(target=fun1)
    t1.start()
""", 10, 13, path="/var/folders/3h/s9r7m0xn1s5gp0s8_jrjnv6h0000gn/T/py-1798S4d")
gd = s.goto_definitions()

if len(gd) > 0:
    script_modules = gd[0]._evaluator.modules
    if len(script_modules) > 0:
        related = '\n    '.join([smod for smod in script_modules if 'py-' not in smod])
    else:
        related = None

    print('''Help on {0}:

NAME
    {3}

{4}

FILE
    {1}::{2}

OTHER MODULES IN THIS FILE
    {5}
'''.format(gd[0].full_name, gd[0].module_path, gd[0].line, gd[0].name, gd[0].docstring(), related))
```
The error message (AttributeError: 'Evaluator' object has no attribute 'modules') is raised by line `script_modules = gd[0]._evaluator.modules`

May be this issue can be reproduced by using latest version (0.12.0) of jedi.
```
>>> import jedi
>>> print(jedi.__version__)
0.12.0
```

**Comment by jkitchin** [comment]
I believe that could be true. It works for me with jedi 0.11, can you just downgrade to that? something like:

pip install jedi==0.11.1

**Comment by olgu** [comment]
Hi,

I managed to fix the issue under `jedi 0.13.1` by replacing the `if` statement from above with the code

```python
if len(gd) > 0:
    script_modules = list(gd[0]._evaluator.module_cache.iterate_modules_with_names())
    if len(script_modules) > 0:
        related = '\\n    '.join([smod[0] for smod in script_modules if 'py-' not in smod])
    else:
        related = None
```

**Comment by jkitchin** [comment]
Thanks for the suggestion. I think I pushed a fix for this in 78d6fcf. Its not super pretty, since I tried to maintain compatibility with multiple versions, but I think it works for 0.11+ now.
